### PR TITLE
v1.13.0 Multiple style fixes

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- Return back previous links for popular locations
- Return back previous links for popular cuisines
- Add link to company website
- Change link for pizza cuisine
- Fix some styles and layout for feedback block
- Fix some styles for country selector
- Remove `target="_blank"` from country selector

Related tickets [WH-988](https://jira.just-eat.net/browse/WH-988), [WH-999](https://jira.just-eat.net/browse/WH-999), [WH-1000](https://jira.just-eat.net/browse/WH-1000), [WH-1001](https://jira.just-eat.net/browse/WH-1001), [WH-989](https://jira.just-eat.net/browse/WH-989), [WH-1002](https://jira.just-eat.net/browse/WH-1002) [WH-990](https://jira.just-eat.net/browse/WH-990)

### CODE REVIEW

**Comment by DamianMullins** [comment]
@dandel10n I think you may need to update your version of Yarn as it appears to be stripping out all of the integrity lines.

File: src/templates/footer/partials/apps-and-social-links.hbs [hunk 1]
```diff
@@ -31,22 +31,8 @@
...
-        <div class="g-col g-span10">
-            <p class="c-footer-feedbackText">
-                {{ i18n "improveOurWebsite" }}
-                <a href="#">{{ i18n "sendFeedback" }}</a>
-            </p>
-        </div>
-    </div>
+    <p class="c-footer-feedbackText">
+        {{ i18n "improveOurWebsite" }}
+        <a href="#" class="o-link--bold o-link--full">{{ i18n "sendFeedback" }}</a>
```
Comment (ashleynolan) [inline src/templates/footer/partials/apps-and-social-links.hbs]: This should probably be a button as it doesn't actually have a link tied to it. Maybe should be done in a different PR though as looks like you didn't change this as part of these updates.

Comment (dandel10n) [reply src/templates/footer/partials/apps-and-social-links.hbs]: I can do it here as it is a small change

Comment (dandel10n) [reply src/templates/footer/partials/apps-and-social-links.hbs]: As discussed, I will rise separate jira for this, because gtm use a tag for click event
